site/docs/guides/operations/kubernetes/healthchecks.md

```diff
@@ -0,0 +1,257 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project 2020. -->
+
+# Healthchecks when running Egeria in Kubernetes
+
+There are various API calls that will check the status of Egeria.
+
+These may be typically used in a Kubernetes environment to check if Egeria is ready to service requests. Here we summarize what is available.
+
+## Defining a Kubernetes health check
+
+See also [Kubernetes docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+
+Kubernetes has 3 types of health checks
+- startup -- to confirm a pod has started
+- readiness -- to confirm a pod is ready. Typically this will then allow requests to be routed here
+- liveness -- to check the pod is still responding to requests in a timely fashion.
+
+Typically pods will be restarted if these health checks do not pass in a specified time period.
+
+Each of these checks can be of several types
+- tcpSocket -- this just checks for an open port.
+- grpc -- issues grpc call (we do no use grpc in egeria)
+- httpGet -- a simple GET. If return is >=200 and <400 it is successful
+- exec -- issues a specified command within the container
+- 
+## Example Egeria API calls
+
+In these examples the [httpie](https://httpie.io) tool will be used as it will print both the response code, and pretty-formatted body by default. Other tools like curl may also be used, but more parsing may be required of the responses.
+
+The examples here were run against the lab charts, using user 'garygeeke'. A simple security plugin is active which restricts user access to api calls.
+
+### Platform
+This checks if the *platform* is available.
+
+#### Platform is not running
```

Comment:
This call is deprecated.  Should be `https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/platform-services/users/admin/server-platform/origin`